src/modules/healtcheck/healtcheck.controller.ts

```diff
@@ -0,0 +1,76 @@
+import { Controller, Get, Version } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { HealtcheckService } from './healtcheck.service';
+
+@ApiBearerAuth()
+@ApiTags('healtcheck')
+//@Controller('healtcheck')
+@Controller('')
+export class HealtcheckController {
+  constructor(private readonly healtcheckService: HealtcheckService) {}
+
+  @Get('/live')
+  @Version('1')
+  @ApiOperation({ summary: 'live' })
+  @ApiResponse({ status: 200, description: 'Check if this api is running'})
+  live(): any {
+    return {
+      status: 200,
+      message: "Online",
+      datetime: new Date().toISOString()
+    };
+  }
+
+  @Get('healtcheck')
```

Comment:
can we standardise the naming conventions: i.e. with a leading `/` or without

Reply:
No slash is cleaner. Slash was added in healthcheck module while fixing routing and I forgot to remote it.